ui/perfherder/graphs/GraphsContainer.jsx

```diff
@@ -556,6 +574,18 @@ class GraphsContainer extends React.Component {
                       />
                     ))}
 
+                  {highlightOtherAlerts &&
+                    highlightOtherAlertsData.length > 0 &&
+                    highlightOtherAlertsData.map((item) => (
+                      <VictoryLine
+                        key={item}
+                        style={{
+                          data: { stroke: 'red', strokeWidth: 1 },
```

Comment:
I will go with a different color for this, red usually means something is wrong.